src/campaign/s3.py

```diff
@@ -0,0 +1,97 @@
+import logging
+import os
+
+import boto3
+from dotenv import load_dotenv
+
+load_dotenv()
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
用 basicConfig 的寫法這樣我印象中應該是沒辦法印出來，會被 Lambda python 本身的 runtime 就有日誌處理器了，所以寫 `logging.basicConfig(level=logging.INFO)` 不會生效 [^1][^2]

```suggestion
import logging
logger = logging.getLogger()
logger.setLevel("INFO") # or logger.setLevel(logging.INFO)
```


[^1]: https://stackoverflow.com/questions/37703609/using-python-logging-with-aws-lambda
[^2]: https://docs.aws.amazon.com/lambda/latest/dg/python-logging.html#python-logging-lib